lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -363,6 +363,31 @@ declare class Complex64Array implements Complex64ArrayInterface {
 	*/
 	get( i: number ): Complex64 | void;
 
+	/**
+	* Returns the first index at which a given element can be found.
+	*
+	* @param searchElement - element to find
+	* @param fromIndex - starting index (inclusive)
+	* @throws searchElement must be a complex number
+	* @throws fromIndex must be an integer
```

Comment:
```suggestion
```

Only in "special" circumstances do we include error annotations. For the first argument, in TypeScript, providing a complex number is the only thing which is possible due to static analysis. For `fromIndex`, providing an integer is not captured by `number`, but we can probably leave off and be fine, in this case. In the end, not including a `fromIndex` throws annotation is a judgment call, and I am okay leaving out here.